packages/networks/features.ts

```diff
@@ -55,6 +55,19 @@ type CosmWasmSocialFeed = {
   feedContractAddress: string;
 };
 
+// CosmWasm Launchpad
+
+const zodCosmWasmLaunchpad = z.object({
+  type: z.literal(NetworkFeature.NFTLaunchpad),
```

Comment:
we should keep the naming consistent and explicit

```suggestion
const zodCosmWasmNFTLaunchpad = z.object({
  type: z.literal(NetworkFeature.CosmWasmNFTLaunchpad ),
```